server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds.Contracts/DownloadApplicationSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Fabric.Description;
```

Comment:
apply the current Resharper settings and reformat all your changes.

Reply:
Added some styles for code formatting.